wolox-books/src/views/BookDetail.vue

```diff
@@ -0,0 +1,101 @@
+<template lang="pug">
+  .main-container
+    navbar
+    .book-container.row
+      img.book-img(:src='book.image_url' alt='Book image')
```

Comment:
This is a very common mistake between frontend developers. The alternative text should describe what the image represents and not what it is literally speaking.
A better example would be that if there's an input used for searching and it has an icon of a lens, the alternative text would be `search` and not `lens`. Or if there's an icon of a house, the alt shouldn't say `house`, but `home` or `homepage`.